certificate/certificate.go

```diff
@@ -201,6 +201,11 @@ func (cert *Certificate) GetDiscoveryNodes() []core.DiscoveryNode {
 	return result
 }
 
+// GetMajorityRule returns majority rule number
```

Comment:
execute `make generate` and add certificate mock changes to pull request

Reply:
fixed